control/control.openSUSE.xml

```diff
@@ -70,6 +70,13 @@ textdomain="control"
         <!-- FATE: #303893, #305588: Default to enabled kdump -->
         <enable_kdump config:type="boolean">true</enable_kdump>
 
+        <!-- Set SELinux permissive mode by default, https://github.com/yast/yast-installation/pull/906#issuecomment-784238549 -->
+        <selinux>
+          <mode>permissive</mode>
+          <configurable config:type="boolean">true</configurable>
+          <patterns>selinux</patterns>
```

Comment:
Note: pending to check that the pattern exists (it does in openSUSE Tumbleweed)

Reply:
As it happens with SLE, there is no a SELinux pattern available yet in Leap 15.3

![Screenshot_sle15sp3_2021-02-26_08:19:24](https://user-images.githubusercontent.com/1691872/109274492-78b47d80-780b-11eb-9b55-ec27b7426b88.png)

Reply:
@jreidinger, as agreed, I force pushed the PR to set SELinux as disabled by default since, as mentioned several times, there is no SELinux pattern (thus, no SELinux policy package) available yet.

Please, review it again if you don't mind.

Reply:
I think it is wrong to mention pattern that does not exist as it cause another problem.

Reply:
@jreidinger, fixed.